### ISSUE TITLE
ENOENT: no such file or directory, unlink 'C:\Users\my\AppData\Local\Temp\.vscode-projects-plus_branch-git-cache.json'

### ISSUE THREAD

**Issue by ronggang** [fabiospampinato/vscode-projects-plus#9]
Hi everyone,
I'm installed Projects+ 1.11.0，VSCode version 1.17, my `vscode_projects.json` is:
```
{
  "projects": [
    {
      "name": "testCode",
      "icon": "code",
      "path": "D:/code/testCode"
    },
    {
      "name": "testCode1",
      "icon": "code",
      "path": "D:/code/testCode1"
    }
  ]
}
```
![tim 20171016092721](https://user-images.githubusercontent.com/8065899/31591679-6f0f9326-b254-11e7-802d-92d229aec390.png)
When I click the refresh button, the following error is displayed:

> ENOENT: no such file or directory, unlink 'C:\Users\my\AppData\Local\Temp\.vscode-projects-plus_branch-git-cache.json'

What is the problem?

**Comment by fabiospampinato** [comment]
Thank for reporting the issue, it will be addressed in the next update.

**Comment by fabiospampinato** [comment]
I've just published v1.11.1, it should fix this.
